modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,150 @@
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. The Operator prevents `br-ex` bridge configuration conflicts by sending a signal to the `configure-ovs.sh` shell script to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+
+
+.Procedure
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `MachineConfig` manifest file and define a customized `br-ex` bridge network configuration similiar to the following example:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+  name: 10-br-ex-worker
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents: <2>
+          source: <base64-encoded_access_key_id>
+          mode: 0644
+          overwrite: true
+          path: /etc/nmstate/openshift/<hostname>.yaml
+# ...
+----
+<1> Specify the `NetworkManager` connection name for the primary network interface.
+<2> For each note in your cluster, specify the hostname path to your node and the base-64-encoded Ignition configuration file data for the machine type.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a `NodeNetworkConfigurationPolicy (NNCP)` manifest file and define a customized `br-ex` bridge network configuration similiar to the following example:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: worker-0-br-ex <1>
+spec:
+  nodeSelector:
+    kubernetes.io/hostname: worker-0
+  desiredState:
+    interfaces:
+    - name: enp2s0 <2>
+      type: ethernet <3>
+      state: up <4>
+      mtu: 1550
+      ipv4:
+        enabled: false <5>
+      ipv6:
+        enabled: false
+    - name: br-ex
+      type: ovs-bridge
+      state: up
+      ipv4:
+        enabled: false
+        dhcp: false
+      ipv6:
+        enabled: false
+        dhcp: false
+      bridge:
+        port:
+        - name: enp2s0 <6>
+        - name: br-ex
+    - name: br-ex
+      type: ovs-interface
+      state: up
+      copy-mac-from: enp2s0
+      mtu: 1550
+      ipv4:
+        enabled: true
+        dhcp: true
+        address:
+        - ip: "169.254.169.2"
+          prefix-length: 29
+      ipv6:
+        enabled: false
+        dhcp: false
+----
+<1> Name of the policy.
+<2> Name of the interface.
+<3> The type of interface. This example creates a bridge.
+<4> The requested state for the interface after creation.
+<5> Disables IPv4 and IPv6 in this example.
+<6> The node NIC to which the bridge is attached.
+
```

Comment:
Somewhere here we need to enumerate the list of changes that are allowed on day 2. I copied the list out of the enhancement into Jira for easy reference: https://issues.redhat.com/browse/OPNET-513?focusedId=24897779&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-24897779